### DISCUSSION TITLE
Full Node Status 'Not Synced'

### DISCUSSION THREAD

**Discussion by sor1706** [Chia-Network/chia-blockchain#5326] [Support]
**Describe the bug**

Full Node Status 'Not Synced'
Wallet 'Not Synced connections 1

Since updating to latest 1.1.5 Chia I cannot Sync I have tried uninstalling rebooting reinstalling tried to manually to connect to Peers closing opening deleting ploting, plotting and nothing i do can make this thing even attempt to Sync. It has been 4 days and have now given up. I have managed to open up an old laptop on the same netwrk and that has attempted to sync straight away on my exisiting account but is really slow.

My PC is a 128gb Ram 3970x Threadripper uising Windows 10

What else can I try to sync this thing?

**Comment by masoudmatrix** [comment]
I have this problem too
Not syncing

**Comment by sor1706** [comment]
Why can you not add a button which forces it to sync or some sort of override. It's clearly too tempremental to leave to it's own devices as you can see from many others reporting the same issue.

**Comment by captainmildred** [comment]
Same issue.

**Comment by doothcrow** [comment]
Yep. Same here. 
It would say 'Not Synced', so I relaunched the software. No luck. Updated to the latest version. No luck. Same error. Lost for what to try now.

**Comment by perrin4869** [comment]
Same boat for about a week ><;;

**Comment by leefarg** [comment]
Same issue for 3 days now.

**Comment by doothcrow** [comment]
After updating 1.1.5 and restarting the app a few times, mine does now say Synced. Yesterdays 2 year ETA is now 5 years, but at least it's doing something!

**Comment by Noodleyman** [comment]
Same, this issue is very frustrating. Am connected to 10+ full nodes successfully, then the status drops to:
![image](https://user-images.githubusercontent.com/10614018/118481740-b7234f80-b70b-11eb-9f40-1029dffaf464.png)

And, can sit here for HOURS before it'll maybe re-sync itself. Plenty of open, valid connections. why won't it sync? It's got a lot worse over the past few days. estimating that over a 24 hour period, 6+ hours are spent disconnected for no reason I can find.. it just won't sync...

This is a critical issue, as not in sync means not farming...

**Comment by SirVladislav** [comment]
Chia 1.1.5  everything was fine yesterday

problems started after reboot chia GUI this morning
trying to connect but no response

2021-05-17T15:14:09.767 full_node full_node_server        : INFO     -> new_peak to peer 183.179.213.148 be4c9923fb02928524319850aea04b6395bedbf5927ce42928b452fdc85d4f69
2021-05-17T15:14:09.770 full_node full_node_server        : INFO     Connection closed: 183.179.213.148, node id: be4c9923fb02928524319850aea04b6395bedbf5927ce42928b452fdc85d4f69
2021-05-17T15:14:09.772 full_node chia.full_node.full_node: INFO     peer disconnected {'host': '183.179.213.148', 'port': 8444}
2021-05-17T15:15:17.353 full_node full_node_server        : INFO     -> handshake to peer 142.114.140.103 57c8a081ecd7be442e6631a71d7ceccb262dac04fed925b8edee96b0f5add179
2021-05-17T15:15:17.479 full_node full_node_server        : INFO     Connected with full_node {'host': '142.114.140.103', 'port': 8444}
2021-05-17T15:15:17.481 full_node full_node_server        : INFO     -> new_peak to peer 142.114.140.103 57c8a081ecd7be442e6631a71d7ceccb262dac04fed925b8edee96b0f5add179
2021-05-17T15:15:17.484 full_node full_node_server        : INFO     Connection closed: 142.114.140.103, node id: 57c8a081ecd7be442e6631a71d7ceccb262dac04fed925b8edee96b0f5add179
2021-05-17T15:15:17.485 full_node chia.full_node.full_node: INFO     peer disconnected {'host': '142.114.140.103', 'port': 8444}
 
and problems with wallet synce on machines where  full node synchronization is OK

**Comment by rfxn** [comment]
If you check your full node connections (chia show -c), you can see that there is a large number of new nodes at low block height and/or those tangibly behind the current block height. This makes reaching block-height consensus difficult as when your node discovers other nodes, none of them are fully synced.

Patience. Do not stop your full node, allow it to continue syncing and catching up with other peers and the problem will start to correct itself. As of the time of this posting, all my full nodes are able to maintain sync whereas last night they were all falling out of sync every 10 minutes.

You can also look to force your full node to peer with a known in-sync node (chia show -a IP:8444).  This is not a silver bullet as in-sync nodes will have significant request load currently.

**Comment by lmbaigit** [comment]
same problem

**Comment by waitkeeper** [comment]
I have this problem too
Not syncing

**Comment by captainmildred** [comment]
I’m still having same problem, have just left it to sync for 3 days to catch up while it syncs/ not syncs. Has now dropped a full 24 hours behind and is getter further away. Any suggestions?

**Comment by PedroMCK** [comment]
the same problem happened to me too! everything was working perfectly but with this last update 11. 5 I can't synchronize anymore!

**Comment by Yamanipanuchi** [comment]
I've been able to resolve this very same issue by running one of the pre-release copies. I hope a new release comes out soon to resolve this issue that so many people seem to be having.

**Comment by perrin4869** [comment]
Switching to main branch solved the issue for me too!

**Reply by leefarg** [reply]
Sorry - don't understand. What do you mean "switching to main branch"?
Thanks!

**Comment by BrandtH22** [comment]
@sor1706 , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
